# Looking at basic summary stats for pipeline incidnets since 2010

* 'Signficiant' and 'non-significant' events
* Events include the following: 
    * Gas distribution
    * Gas Transmission and Gathering
    * Hazardous liquid
    
    **NOTE:** Liquified Natural Gas incidents are not included in this analysis (32 total LNG incidents since 2011)
    
[Data source](https://www.phmsa.dot.gov/data-and-statistics/pipeline/source-data)
Data downloaded on: 

In [1]:
import pandas as pd
import numpy as np
import os

df = pd.read_csv('../data/processed/pipeline_incidents_2010_present_all.csv')


/Users/jessieblaeser/.pyenv/versions/3.10.0/lib/python3.10/site-packages/pandas/compat/__init__.py:124: UserWarning: Could not import the lzma module. Your installed Python is incomplete. Attempting to use lzma compression will result in a RuntimeError.
  warnings.warn(msg)
/Users/jessieblaeser/.pyenv/versions/3.10.0/lib/python3.10/site-packages/IPython/core/interactiveshell.py:3444: DtypeWarning: Columns (1,15,16,18,19,20,21,22,25,26,27,28,29,30,31,32,33,36,56,57,58,63,64,65,72,73,75,76,77,78,84,85,87,89,92,93,94,95,101,102,103,104,105,106,108,109,116,120,148,149,151,152,154,171,172,173,193,194,197,198,205,209,210,214,217,218,219,221,222,223,225,228,230,231,233,234,238,239,240,250,251,252,256,257,261,262,263,265,266,268,269,276,277,278,279,280,281,286,287,290,293,294,295,296,297,300,301,303,304,305,306,307,310,311,313,314,315,316,317,318,319,320,321,322,324,325,326,327,329,330,331,332,333,334,335,336,337,340,341,342,343,344,345,347,348,351,354,355,356,357,359,360,361,362,363,365,366,3

In [2]:
# Total cost adjusted for inflation
df['total_cost_current'].sum()

7994914265.472826

In [3]:
# Total injuries 
df.injure.sum()

716

In [4]:
# Total deaths
df.fatal.sum()

156

In [5]:
# Total incidents
df.shape

(7499, 826)

# Conclusions: 
* There have been 7,499 pipeline incidents from 2010 to date of data download
* Total cost of pipeline incidents since 2010 adjusted for inflation = ~ 
$7,994,914,265
* Pipeline incidents since 2010 have injured 716 people
* Pipeline incidents since 2010 have killed 156 people
